File: JJB/231206/한_번만_등장한_문자.py
def solution(my_string):
    answer = ''
    bindo = {}
    
    for s in my_string:
        try:
            bindo[s] += 1
        except:
            bindo[s] = 1
    
    bindo = dict(sorted(bindo.items(), key=lambda s : s[0]))
    
    for key, value in bindo.items():
        if value == 1:
            answer += key
    
    return answer

File: JJB/231206/한_번만_등장한_문자.cpp
#include <string>
#include <vector>
#include <unordered_map>
#include <algorithm>
#include <iterator>

using namespace std;

string solution(string s) 
{
    string answer = "";
    unordered_map<char, int> char_count;
    
    for(const char &c : s)
        char_count[c]++;
    
    vector<int> unique_char;
    copy_if(s.begin(), s.end(), back_inserter(unique_char), [&char_count](char c){ return char_count[c]==1; });
    
    sort(unique_char.begin(), unique_char.end());
    
    answer = string(unique_char.begin(), unique_char.end());
    
    return answer;
}